src/edgepi/calibration/edgepi_eeprom.py

```diff
@@ -1,5 +1,7 @@
 '''Helper class to access on board eeprom'''
 
+# pylint: disable=no-name-in-module
```

Comment:
what do you need this for?

Reply:
https://github.com/protocolbuffers/protobuf/issues/10372

issue with the pylint not recognizing the module name of protobuf generated object

Reply:
@sjpark608 include issue link as comment in code next to pylint disable.

Reply:
df3486b99fb2b6ed549393ac55c7022e3d27dbe0